src/wasm/kotlin/java/arcs/jvm/JvmRuntimeClient.kt

```diff
@@ -0,0 +1,38 @@
+package arcs
+
+actual object RuntimeClient {
```

Comment:
Consider KDoc for this,  explaining why the methods are all No-Ops?    Additionally, for No-Op void/unit methods, you can do `fun functionName(arg1: Foo,...) = Unit` rather than  open/close braces.

Reply:
I suspect these are all TODOs, in which case, I'd suggest throwing UnsupportedOperationExceptions, and adding TODOs

Reply:
That's a good tip. What a cool Kotlin feature.